client/app/states/designer/blueprints/list/list.state.js

```diff
@@ -86,13 +86,6 @@
       visibilityNames.push(item.name);
     }
 
-    /* This notification 'splice' code doesn't work.  Splice needs a third argument, the items to splice in
```

Comment:
@jeff-phillips-18 Jeff, this code does work, splice doesn't need more than 2 arguments... it clears the array without destroying any references :).

You can think of it as $rootScope.notifications.data = []; except you can be sure that if you've done foo = $rootScope.notifications.data before, then even foo will be empty (and still point to the same array).